public/js/controllers/charts_controller.js

```diff
@@ -319,14 +342,14 @@ export default class extends Controller {
 
       case 'chainwork': // Total chainwork over time
         d = zipYvDate(data)
-        assign(gOptions, mapDygraphOptions(d, ['Date', 'Cumulative Chainwork (exahash)'],
-          false, 'Cumulative Chainwork (exahash)', 'Date', undefined, true, false))
+        assign(gOptions, mapDygraphOptions(d, ['Date', 'Cumulative Chainwork (exahash/s)'],
+          false, 'Cumulative Chainwork (exahash/s)', 'Date', undefined, true, false))
         break
 
       case 'hashrate': // Total chainwork over time
         d = zipYvDate(data)
-        assign(gOptions, mapDygraphOptions(d, ['Date', 'Network Hashrate (terahash/s)'],
-          false, 'Network Hashrate (terahash/s)', 'Date', undefined, true, false))
+        assign(gOptions, mapDygraphOptions(d, ['Date', 'Hashrate'], false, 'Hashrate (per seconds)', 'Date', undefined, false, false))
```

Comment:
`Hashrate (per seconds)` doesn't make too much sense since it's already described as a rate.  I suggest having this be just `Hashrate` or `Hashrate (hashes per second)` if you feel that adds anything, but I don't think it does.